ci_build/azure-pipelines/mshost.yaml

```diff
@@ -3,227 +3,269 @@
 #######
 
 jobs:
-- job: Linux
```

Comment:
why is every line changed? it's hard to review

Reply:
I format this file, only added lines: 230-271